src/documentation/pages/components-wip/footer/examples/_external-brightspeed.vue

```diff
@@ -0,0 +1,236 @@
+<template lang="pug">
+  <ComponentExample title="External" id="external-brightspeed" :tabs="exampleTabs">
```

Comment:
It does not look like the same footer.

![image](https://user-images.githubusercontent.com/86958774/176478601-3ad25bdc-f7ed-4c24-94b7-2297c4a5403a.png)

Reply:
@josecarlosgonzalezv I did observe for brightspeed, it is showing up Lumen types but when i checked the code, it is same as the code which is there in the current prod documentation. Not sure but for some reason brightspeed theme dependent css is not getting updated.